craft_parts/packages/deb.py

```diff
@@ -515,7 +516,7 @@ def _get_packages_marked_for_installation(
     @classmethod
     def download_packages(cls, package_names: list[str]) -> None:
         """Download the specified packages to the local package cache area."""
-        logger.debug("Downloading packages: %s", " ".join(package_names))
+        logger.debug("downloading packages using apt: %s", " ".join(package_names))
```

Comment:
nit: all log lines across our libraries start with a capital letter. Is there a reason you're using lowercase in this PR?